.github/workflows/test.yml

```
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v4
+      with:
+        version: 8
+
+    - name: Install dependencies
+      run: pnpm install
+
```

```suggestion

    - name: Install dependencies
      run: pnpm biome check

```

tho eslint is also in here I guess? do we need both or can we get rid of eslint?

---

I'm not using eslint in this. I want to do another PR where I set up knip and clean up the dev dependencies

---

This is said [issues](https://github.com/coder/parameters-playground/issues/46) and I think I'll make it a fast follow to this PR. Even including the lint check in the PR is going to bloat the diff